ArtemShchepets/src/main/java/week1/interfaces/IAppDB.java

```diff
@@ -0,0 +1,38 @@
+package week1.interfaces;
+
+import week1.model.Bill;
+import week1.model.Seller;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+public interface IAppDB {
+
+    Logger getLogger();
```

Comment:
Do not need this. Create class MyLogger with method: public void writeLog(String message)